src/App.jsx

```diff
@@ -28,17 +28,17 @@ function App() {
     <Router>
       <section className="header__section">
         <header className="header__section__container">
-          <h1 className="header__title">Bookstore</h1>
-          <nav>
+          <nav className="header__navbar__container">
             <ul className="header__links__container">
+              <h1 className="header__title">Bookstore CMS</h1>
               <Link
                 to="/"
                 className={
-                  activeLink === 'home' ? 'activeLink' : 'header__link__item'
+                  activeLink === 'books' ? 'activeLink' : 'header__link__item'
                 }
-                onClick={() => handleClick('home')}
+                onClick={() => handleClick('books')}
               >
-                Home
+                Books
               </Link>
```

Comment:
- [ ] Great job styling your bookstore web app so beautifully 🥰. However, I noticed you missed little details of the design that when added your design is going to look perfect 👌. Kindly consider the below suggestions to help you add these missing details:

 - Kindly consider adding the user `profile`  icon at the far end of your `nav` as shown on the original design given in the project requirements. See below for more details:
 
<table>
<tr>
<th align=“middle”>Original design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="863" alt="Screenshot 2023-08-24 at 17 23 12" src="https://github.com/Felipe-Perez-Ferraro/bookstore/assets/107192076/c4448651-50b6-4de4-8cd4-db5f69754d5e">
</td>
<td>
<img width="1403" alt="Screenshot 2023-08-24 at 17 22 45" src="https://github.com/Felipe-Perez-Ferraro/bookstore/assets/107192076/79ca1d01-1b0e-48ba-a7b7-9db52ce43ecb">
</td>
</tr>
</table>